mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -436,6 +439,40 @@ export class MetricsPlotPanel extends React.Component {
     };
     this.updateUrlState({ layout: mergedLayout, lastLinearYAxisRange });
   };
+  
+  convertJsonToCsv = () => {
+    const metrics = this.getMetrics();
+    //const parsedJson = JSON.parse(metrics);
+    //console.log("Parsed JSON: " + parsedJson)
+    const selectedMetricKeys = this.getUrlState().selectedMetricKeys;
+    console.log("selected metrics: " + selectedMetricKeys);
+    const heading = Object.keys(metrics[0].history[0]).join(",") + "\n"
```

Comment:
Could you take a video / screenshot of what a subset selection of `selectedMetricKeys` are in which you're selecting 2 or 3 of perhaps 10 metrics and illustrating what the .csv output will be (including the named headers)

Reply:
sure thing, please find below:

**- scenario 1 - one metric selected:**

![Screenshot 2022-06-14 at 12 54 47](https://user-images.githubusercontent.com/97888102/173561676-4ba7b140-e36c-443e-957e-6b25739e3260.png)

output CSV: 

![Screenshot 2022-06-14 at 12 57 35](https://user-images.githubusercontent.com/97888102/173562017-a7320f63-f5b9-4e08-a6a0-0378df26aaae.png)

[metrics.csv](https://github.com/mlflow/mlflow/files/8899116/metrics.csv)

**- scenario 2 - two metrics selected:**

![Screenshot 2022-06-14 at 12 55 13](https://user-images.githubusercontent.com/97888102/173561792-9bb65c2e-2b40-4fc1-b4b4-03ac79fde042.png)

output CSV: 

![Screenshot 2022-06-14 at 12 57 46](https://user-images.githubusercontent.com/97888102/173562042-cde78f1b-29eb-4c06-8d40-3944cc81e084.png)


[metrics (1).csv](https://github.com/mlflow/mlflow/files/8899120/metrics.1.csv)